islandora_video_streaming_kaltura.module

```diff
@@ -81,4 +80,41 @@ function islandora_video_streaming_kaltura_process_islandora_video($variables =
   }
 }
 
+/*
```

Comment:
These functions from kaltura shouldn't be part of the panopto PR.  This should be resolved by merging master into Q-Panopto.